ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Overview/PlatformCvesOverviewPage.tsx

```diff
@@ -45,7 +45,7 @@ function PlatformCvesOverviewPage() {
     const querySearchFilter = searchFilter;
     const isFiltered = getHasSearchApplied(querySearchFilter);
 
-    const isViewingSnoozedCves = querySearchFilter['CVE Snoozed'] === 'true';
+    const isViewingSnoozedCves = querySearchFilter['CVE Snoozed']?.[0] === 'true';
```

Comment:
Now that the "Show snoozed CVEs" toggle is available in the header, I was able to test with the actual logic and found that this doesn't work as expected. Including the fix for both Platform and Node in this PR.